boards/arm/mec15xxevb_assy6853/dts_fixup.h

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
This file is not necessary if you use the devicetree names in the driver.

Reply:
Removed.